### DISCUSSION TITLE
Should we consider alternatives to monospace fonts?

### DISCUSSION THREAD

**Discussion by ifokkema** [HGVSnomenclature/hgvs-nomenclature#160] [HGVS presentation]
A curator recently let me know that they get confused by the use of the monospaced font on the website because they think spaces are included in the variant descriptions. The example given was `NG_012232.1(NM_004006.2):c.93+1G>T` (taken from the DNA substitution page). The curator indicated she felt there were spaces around the period after the prefix and before the positions. This is due to the monospace font used to describe variants on the website. It's possible that different browsers or systems use a different font and that the problem will be worse in some systems than others.

Do we feel more people might have this issue? Does this mean we should perhaps consider a different font? It would still be nice to easily highlight variant descriptions on the site.

**Comment by ahwagner** [comment]
I am not sure this is a common problem and would be hesitant to make font changes on this feedback.

However, AFAIK white space is not used in variant descriptions, and is an implied delimiter for use of the HGVS nomenclature within a body of text. Should we simply state as much in our General recommendations section?

**Reply by jfjlaros** [reply]
I am not sure about explicitly adopting white space as a delimiter, it may have implications on how lenient variant normalisation procedures can be (all tools I know filter out white space).

I will give it some more thought.

**Reply by reece** [reply]
There is a quasi-recommendation to write protein variants with a space. From https://archive.hgvs-nomenclature.org/recommendations/checklist/:

> when descriptions at protein level are given in the text, upon first appearance, use a format like “c.76G>T (p.(Gly26Cys), RNA not analysed)” or “c.76G>T (r.76g>u p.Gly26Cys)”

(I'm citing the archive here because we lifted this block over slightly awkwardly in the new pages.)

The point is that we do recommend spaces in these compound-type variants. 

I agree that we shouldn't change the font based on one person's feedback. Let's wait to see if anyone else reports this.

**Reply by jfjlaros** [reply]
I would say that only the parts highlighted in monospace in "`c.76G>T` (`p.(Gly26Cys)`, RNA not analysed)" and "`c.76G>T` (`r.76g>u` `p.Gly26Cys`)" are covered by the nomenclature, the rest of the text is part of the English language.

**Reply by ifokkema** [reply]
Whitespace is indeed never used in variant descriptions, but a single space also doesn't seem much of a delimiter. In the docs, I sometimes added additional spacing to separate multiple variant descriptions.

Heidi's [recent comment](https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/18#discussioncomment-8654663) on the parentheses subject also mentioned her request to use parentheses as separators, although she requested it to become part of the variant description itself as a separator, which doesn't sound like a good idea to me.


> I would say that only the parts highlighted in monospace in "`c.76G>T` (`p.(Gly26Cys)`, RNA not analysed)" and "`c.76G>T` (`r.76g>u` `p.Gly26Cys`)" are covered by the nomenclature, the rest of the text is part of the English language.

I agree, which is also why I would reject the recommendation of parentheses around protein descriptions to separate them from DNA descriptions (Heidi's second unnumbered point in the above-mentioned comment).
